tests/UI/pages/FO/product/index.js

```diff
@@ -2,11 +2,11 @@ require('module-alias/register');
 const FOBasePage = require('@pages/FO/FObasePage');
 
 class Product extends FOBasePage {
-  constructor() {
+  constructor() {d
```

Comment:
```suggestion
  constructor() {
```

Reply:
Oula the keyboard miss touch sry ;) Corrected.